Multisig/Logic/Ledger/LedgerController.swift

```diff
@@ -30,14 +30,27 @@ class LedgerController {
         }
         let path = "44'/60'/0'/0/\(index)"
         let command = getAddressCommand(path: path)
-        bluetoothController.sendCommand(device: device, command: command) { [weak self] result in
+
+        // We don't use [weak self] with private methods not to capture LedgerController in a caller
+        bluetoothController.sendCommand(device: device, command: command) { result in
             switch result {
             case .success(let data):
-                guard let addressString = self?.parseGetAddress(data: data),
+                guard data.count == 109,
+                      Int(data[5]) == 65, // public key length
```

Comment:
might be better to define constants with meaningful names for those values

Reply:
I think this is not necessary for low-level code and it is readable